src/redux/actions/dictionaries/dictionaryActionCreators.js

```diff
@@ -57,7 +57,7 @@ export const createDictionaryUser = data => dispatch =>
         ),
       )})
     .catch(error => {
-      error.response ? notify.show(`${error.response.data.__all__[0]}`, 'error', 6000) : 
```

Comment:
Are all these changes related to the ticket?

Reply:
I have reverted unnecessary changes.

Reply:
All of them? In other words, did you go again through the entire commit and ask if each change was necessary for the purposes of this ticket?